### PR TITLE
Add new test suite sdk+desktop

### PR DESCRIPTION
**Pull request by mloviska**
- Related ticket: [[JeOS][sle15sp2] identify all test cases requiring registration against SDK module](https://progress.opensuse.org/issues/59199)
- Verification runs:
  * [sle-15-SP2-JeOS-for-kvm-and-xen-x86_64-Build2.62-jeos-sdk+desktop@64bit-virtio-vga](http://eris.suse.cz/tests/2857) 

 I have local connectivity issues to get proper HyperV and xen runs.

### CODE REVIEW

**Comment by mloviska** [comment]
@Vogtinator @gmoro Any ideas what else we can add to test SDK and DESKTOP modules?

**Comment by gmoro** [comment]
> @Vogtinator @gmoro Any ideas what else we can add to test SDK and DESKTOP modules?

Not sure from the top of my head, but please bring up the topic next meeting so we can have a quick chat.

File: tests/toolchain/gcc_compilation.pm [hunk 1]
```diff
@@ -27,6 +28,8 @@ sub run {
     # poo#33376: added to investigate OOM
     assert_script_run 'free -m';
     save_screenshot;
+    sleep 3;
```
Comment (Vogtinator) [inline tests/toolchain/gcc_compilation.pm]: Why?

File: tests/toolchain/install.pm [hunk 2]
```diff
@@ -55,7 +55,7 @@ sub run {
             add_suseconnect_product("sle-module-development-tools");
         }
         zypper_call 'in -t pattern devel_basis';
-        zypper_call 'in gcc-fortran';    # from Base System Module
+        zypper_call((is_jeos) ? 'in gcc-fortran bzip2' : 'in gcc-fortran');    # from Base System Module
```
Comment (Vogtinator) [inline tests/toolchain/install.pm]: Just always install it - if it's already installed, that's a noop.

File: tests/toolchain/gcc_compilation.pm (hunk 1 above)
Comment (mloviska) [reply tests/toolchain/gcc_compilation.pm]: I cannot find the occurrence atm, but I have seen that rarely screenshotting takes a little more time resulting in an inconsistent screenshot. Thus, there's a chance that openqa would save an image of an already/partially cleared screen, which is useless. But at second glance and thought, we might even get rid of L#30 and L#31. 
If the content of the ``free`` command's output has any value for us, we can just simply redirect it to the serial console. wdyt ?

Comment (Vogtinator) [reply tests/toolchain/gcc_compilation.pm]: The referenced poo# seems resolved and old, I'm in favor of just dropping that.

Comment (mloviska) [reply tests/toolchain/gcc_compilation.pm]: Make sense, even that particular was dropped from the ``gcc_fortran_compilation.pm`` module.
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4709/files

File: tests/toolchain/gcc_compilation.pm [hunk 3]
```diff
@@ -14,19 +14,17 @@ use base "opensusebasetest";
...
     assert_script_run('cd ./gawk-4.1.4');
     assert_script_run('./configure 2>&1 | tee /tmp/configure.log; if [ ${PIPESTATUS[0]} -ne 0 ]; then false; fi',                    600);
     assert_script_run('make -j$(getconf _NPROCESSORS_ONLN) 2>&1 | tee /tmp/make.log; if [ ${PIPESTATUS[0]} -ne 0 ]; then false; fi', 3600);
     assert_script_run('./gawk \'{ print }\' /etc/hostname');
     save_screenshot;
-
-    # poo#33376: added to investigate OOM
-    assert_script_run 'free -m';
-    save_screenshot;
+    clear_console;
```
Comment (Vogtinator) [inline tests/toolchain/gcc_compilation.pm]: Why is this necessary?

Comment (mloviska) [reply tests/toolchain/gcc_compilation.pm]: A successor test case can start with a clear screen. 
http://eris.suse.cz/tests/2865#step/gcc_fortran_compilation/1

Just a convenience for future test case reviewers, but definitely not necessary.
